handbook/engineering/search/iteration_log.md

```diff
@@ -36,7 +36,16 @@ This document contains the goals and work log for the search team's [2-week iter
 - **Outcomes:**
     - We have a huge debt in parts of our search code and the state is unhealthy. I have been tracking a backlog of issues to refactor and fix re: deleting old parser code, fixing commit search issues. I will be selectively tackling [bug](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+author%3Arvantonder+label%3Ateam%2Fsearch+label%3Abug) and [refactor](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+author%3Arvantonder+label%3Ateam%2Fsearch+label%3Arefactor) issues to make things more sane.
 - **Work log:**
-    - YYYY-MM-DD: $UPDATE    
+    - YYYY-MM-DD: $UPDATE
+
+### Streaming search
+
+- **Owner(s):** Keegan, Juliana, Rob
+- **Outcomes:**
+    - We have high-fidelity designs for streaming search.
+    - High-fidelity designs are implemented (TODO: assess viability of this outcome when Juliana is back).
```

Comment:
@keegancsmith are there any outcomes I'm missing here? Any backend work you were intending to tackle this iteration?

Reply:
Will update, was intending to add some real streaming just for text search this iteration.